docs/dashboard.rst

```diff
@@ -0,0 +1,28 @@
+.. _dashboard:
+
+=========================================================
+Rackspace Technology customer portal solutions dashboard
```

Comment:
I don't know that we've used the "solutions dashboard" terminology in customer-facing comms before. @grizzlydev  @lfielder Any input on what we should call this in customer-facing material?

Reply:
When I reached out to the CSMs for feedback, I called it "My Accounts" because that's how we refer to it in Pilot/ Global Nav. What do you think about that @bradgignac @lfielder ?

Reply:
Or maybe "account dashboard", or "account hub", "account navigator"

Reply:
changed to my account dashboard